Jenkinsfile

```diff
@@ -50,15 +61,19 @@ pipeline {
           steps {
             dir('xscope_fileio') {
               withTools(params.TOOLS_VERSION) {
-                installDependencies()
+                createVenv("requirements.txt")
+                withVenv {
+                  sh "pip install -e xtagctl/" // xtagctl before requirements.txt
```

Comment:
Comment doesn't add anything, it should either explain why or be removed - we can already see from the code that xtagctl is installed before requirements.txt